app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from ..player import Player
+
+
+class Dwarf(Player):
```

Comment:
The `Dwarf` class does not implement all abstract methods from the `Player` class, which is why it should inherit `ABC` as well.